src/tpm2/SymmetricTest.h

```diff
@@ -84,7 +84,7 @@ const SYMMETRIC_TEST_VECTOR   c_symTestValues[NUM_SYMS + 1] = {
       dataOut_AES128_CFB, dataOut_AES128_ECB}},
 #endif
 #if ALG_AES && AES_192
-    {TPM_ALG_AES_, 192, key_AES192, 16, sizeof(dataIn_AES192), dataIn_AES192,
+    {TPM_ALG_AES, 192, key_AES192, 16, sizeof(dataIn_AES192), dataIn_AES192,
```

Comment:
Do you need AES_192? Iirc I left this in there on purpose since libtpms doesn't support it in its default config.